test/ngSanitize/sanitizeSpec.js

```diff
@@ -266,14 +268,25 @@ describe('HTML', function() {
     });
   });
 
+  // See https://github.com/cure53/DOMPurify/blob/a992d3a75031cb8bb032e5ea8399ba972bdf9a65/src/purify.js#L439-L449
```

Comment:
Yes, it's mostly to know if/when to remove it. But I commented in a wrong place, the tests should stay forever as they are, such comments belong more in source.